gpt4all-chat/chatllm.cpp

```diff
@@ -170,7 +222,7 @@ bool ChatLLM::loadModel(const ModelInfo &modelInfo)
 #endif
         delete m_llModelInfo.model;
         m_llModelInfo.model = nullptr;
-        emit isModelLoadedChanged(false);
+        emit modelLoadingPercentageChanged(std::numeric_limits<float>::min());
```

Comment:
I think there should be a comment explaining the significance of negative infinity here.

Reply:
This is not negative infinite. It is the smallest possible positive floating point value closest to zero.

Reply:
Oops, sometimes I forget that FLT_MIN has nothing to do with INT_MIN. I'm thinking there should be a brief comment like `// small nonzero value` to make it clear that it is arbitrary.

Reply:
Added comments.